index.html

```diff
@@ -4,71 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio-Setup-Mobile-V</title>
+<!-- PAGE  TITLE -->
+    <title>Martin Pineda's Portfolio</title>
     <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-    <!-- Header section -->
-    <header class="menu-line">
+<!-- BACKGROUND & NAVIGATION MENU -->
+    <header class="header-desktop menu-line">
+        <div class="top-figures-desktop">
+        </div>
+        <div class="direction-desktop">
+            <a href="!">Portfolio</a>
+            <a href="!">About</a>
+            <a href="!">Contact</a>
+        </div>
         <div>
-            <nav>
+            <nav class="container-menu-box">
                 <a href="!" class="menu-box">
                     <img class="menu-icon" src="images/Icon - Menu.svg">
                 </a>
             </nav>
         </div>
     </header>
-    <!-- Main content section -->
-    <section id="head-line">
-        <div class="info">
-            <div class="frame box">
-                <h1 id="title">Hey there.<br>I'm Martin.</h1>
-                <h2 id="title2">I'm a software developer</h2>
-            </div>
-            <!-- Support section -->
-            <p class="frame2">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don't hesitate to contact me.
+<!-- TOP SECTION -->
+    <section class="info-desktop" id="head-line">
+        <div class="info frame">
+            <h1 id="title">Hey there.<br class="jump">I’m Martin.</h1>
+            <h2 id="title2">I’m a software<br class="jump"> developer</h2>
+            <p class="frame2">I can help you build a product, feature or website Look through some of my work and<br> experience! If you like what you see and have a project you need coded,<br> 
+                    don't hesitate to contact me.
```

Comment:
- [ ] You refactoring on desktop should still maintain the mobile design you have. This is the whole idea behind a responsive web app, to be flexible and have a nice UI across differenet screen sizes."
- Kindly adjust the structure of this `header` section to replicate the figma design at the mobile view of `375`.  Major chnages happen to be the background image, and the hanburger button which needs to be righly placed. 
- Using the right `CSS` selectors sech as flecbox on this view port will fix this 👍 

## Your view:
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/5d126c3a-48ad-4ebd-b80a-e2c226ee64d0)

## Figma 
![image](https://github.com/AztkMarty/Portfolio-Setup/assets/50990925/db32c9ab-403e-4101-8ca8-50f7c1977116)